.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on: push
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        uses: jrl-umi3218/github-actions/install-dependencies@master
+        with:
+          compiler: clang
+          ubuntu: |
+            apt: libeigen3-dev
+          macos: |
+            brew: eigen
+      - name: CMake
+        run: cmake -S . -B build -DCMAKE_CXX_COMPILER=$(which clang++)
+      - name: Build hyped
+        run: cd build; make -j
```

Comment:
[`working_directory`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun) might be slightly more syntactic than `cd` but this is just very nitpicky ;)